src/src.pro

```diff
@@ -142,8 +142,9 @@ win32 {
 }
 mac {
     RC_FILE = macapp.icns
-    INCLUDEPATH += -I/usr/local/include \
-                   -I/usr/local/opt/sqlite/include
+    QT+= macextras
+    INCLUDEPATH += /usr/local/include \
```

Comment:
Hmmm, this may have been why I wasn't able to build with SQLCipher support on OSX 10.9 a while ago (but didn't bother looking into it :/).